api/src/service/RpcClient.ts

```diff
@@ -204,13 +212,21 @@ export class RpcClient {
         const plainStringToDecrypt = obj.data.json;
         if (config.encryptionKey) {
           if (typeof plainStringToDecrypt !== "string") {
-            throw new Error(
-              "failed to decrypt events: expected string, but got other type instead. This error usually occurs when you set the ENCRYPTION_KEY at an unencrypted blockchain. To resolve this error, unset the env var ENCRYPTION_KEY.",
+            return new RpcError(
+              500,
+              "Failed to decrypt events: expected string, but got other type instead. This error usually occurs when you set the ENCRYPTION_KEY at an unencrypted blockchain. To resolve this error, unset the env var ENCRYPTION_KEY.",
+              {},
+              "",
             );
```

Comment:
This is how the error looks:
![image](https://user-images.githubusercontent.com/59825636/128026468-33ca99aa-4075-474a-b18d-3c8fae3ef17d.png)